src/dnscap.1.in

```diff
@@ -53,6 +53,7 @@
 .Op Fl E Ar datetime
 .Op Fl P Ar plugin.so
 .Op Fl U Ar str
+.Op Fl q Ar unsigned int
```

Comment:
The phrasing `unsigned int` is not very user friendly, use something like `nth` instead.